peps/pep-0779.rst

```diff
@@ -0,0 +1,174 @@
+PEP: 779
```

Comment:
Any reason not to use 776, the next in sequence?
```suggestion
PEP: 776
```

Reply:
I saw 777 being used and 778 in the PRs, so I assumed 779 was the next one. Do you want me to renumber it?

Reply:
It's fine, you've already opened the discussion, we can fill the gap later.

777 was used out of sequence (re: https://github.com/python/peps/pull/3786#issuecomment-2122743514).

Tip: this checks both published PEPs and pending PRs:

```console
❯ pip install pepotron
❯ pep next
Next available PEP: 776
```